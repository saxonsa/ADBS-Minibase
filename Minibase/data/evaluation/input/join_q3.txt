Q(x, y, z, u, v, w) :- R(x, y, z), S(u, v, w)
