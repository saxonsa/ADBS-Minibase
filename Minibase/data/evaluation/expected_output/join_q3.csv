1, 9, 'adbs', 1, 'smith', 8
1, 9, 'adbs', 2, 'anka', 2
1, 9, 'adbs', 4, 'rhcp', 3
1, 9, 'adbs', 5, 'bowie', 4
1, 9, 'adbs', 8, 'pistols', 5
2, 7, 'anlp', 1, 'smith', 8
2, 7, 'anlp', 2, 'anka', 2
2, 7, 'anlp', 4, 'rhcp', 3
2, 7, 'anlp', 5, 'bowie', 4
2, 7, 'anlp', 8, 'pistols', 5
4, 2, 'ids', 1, 'smith', 8
4, 2, 'ids', 2, 'anka', 2
4, 2, 'ids', 4, 'rhcp', 3
4, 2, 'ids', 5, 'bowie', 4
4, 2, 'ids', 8, 'pistols', 5
5, 0, 'sdp', 1, 'smith', 8
5, 0, 'sdp', 2, 'anka', 2
5, 0, 'sdp', 4, 'rhcp', 3
5, 0, 'sdp', 5, 'bowie', 4
5, 0, 'sdp', 8, 'pistols', 5
8, 1, 'mlpr', 1, 'smith', 8
8, 1, 'mlpr', 2, 'anka', 2
8, 1, 'mlpr', 4, 'rhcp', 3
8, 1, 'mlpr', 5, 'bowie', 4
8, 1, 'mlpr', 8, 'pistols', 5
8, 9, 'rl', 1, 'smith', 8
8, 9, 'rl', 2, 'anka', 2
8, 9, 'rl', 4, 'rhcp', 3
8, 9, 'rl', 5, 'bowie', 4
8, 9, 'rl', 8, 'pistols', 5
8, 9, 'ppls', 1, 'smith', 8
8, 9, 'ppls', 2, 'anka', 2
8, 9, 'ppls', 4, 'rhcp', 3
8, 9, 'ppls', 5, 'bowie', 4
8, 9, 'ppls', 8, 'pistols', 5
